apis/bases/dataplane.openstack.org_openstackdataplanedeployments.yaml

```diff
@@ -65,12 +65,6 @@ spec:
                   type: string
                 minItems: 1
                 type: array
-              preserveJobs:
```

Comment:
I'm not sure if we should be changing the API at this stage of development though. We might want to consider actually making it functional and then dropping it in a future release.

Reply:
we just dropped the entire OpenStackAnsibleEE CRD though. I don't think we need to hold to a strict contract with deprecation period yet. Especially given this wasn't used.